doc/030_preparing_a_new_repo.rst

```diff
@@ -693,3 +693,55 @@ On MSYS2, you can install ``winpty`` as follows:
     $ pacman -S winpty
     $ winpty restic -r /srv/restic-repo init
 
+
+Group accessible repositories
+*****************************
+
+Since restic version FIXME local and SFTP repositories can be made
+accessible to memebers of a system group. To control this we have to change
+the group permissions of the top-level ``config`` file and restic will use
+this as a hint to determine what permissions to apply to newly created
+files. By default ``restic init`` sets repositories up to be group
+inaccessible.
+
+In order to give group members read-only access we simply add the read
+permission bit to all repository files with ``chmod``:
+
+.. code-block:: console
+
+    $ chmod -R g+r /srv/restic-repo
+
+.. TODO: Windows command?
+
+This serves two purposes: 1) it sets the read permission bit on the
+repository config file triggering restic's logic to create new files as
+group accessible and 2) it actually allows the group read access to the
+files.
+
+.. note:: By default files on Unix systems are created with a user's
+          primary group as defined by the gid field in ``/etc/passwd``. See
+          `passwd(5)
+          <https://manpages.debian.org/latest/passwd/passwd.5.en.html>`_.
+
+To manage group membership on Linux systems ``usermod`` can be used. To
```

Comment:
I'm not entirely sure what this paragraph tries to tell me. I'd just leave it out and mention that posix ACLs are an alternative to using setgid.

Reply:
I'm just trying to account for users that aren't super familiar with group managment so I just wanted to drop the names of the relevant tools in here somewhere so they can find docs on this stuff. I've re-arranged this paragraph to be a not eat the end now that should make the flow more smooth.